packages/draft-js-import-element/src/stateFromElement.js

```diff
@@ -298,10 +335,25 @@ class ContentGenerator {
     let block = this.blockStack.slice(-1)[0];
     let style = block.styleStack.slice(-1)[0];
     let entityKey = block.entityStack.slice(-1)[0];
-    style = addStyleFromTagName(style, tagName, this.options.elementStyles);
-    if (ElementToEntity.hasOwnProperty(tagName)) {
-      // If the to-entity function returns nothing, use the existing entity.
-      entityKey = ElementToEntity[tagName](this, tagName, element) || entityKey;
+    let {customInlineFn} = this.options;
+    let customInline = customInlineFn ? customInlineFn(tagName, element, this.inlineCreators) : null;
+    if (customInline != null) {
+      switch (customInline.type) {
+        case 'STYLE': {
+          style = style.add(customInline.style);
+          break;
+        }
+        case 'ENTITY': {
+          entityKey = customInline.entityKey;
+          break;
+        }
+      }
+    } else {
+      style = addStyleFromTagName(style, tagName, this.options.elementStyles);
+      if (ElementToEntity.hasOwnProperty(tagName)) {
+        // If the to-entity function returns nothing, use the existing entity.
+        entityKey = ElementToEntity[tagName](this, tagName, element) || entityKey;
```

Comment:
in the case of `customInlineFn` returning a `style`, this code won't execute, is that okay?

Reply:
👌